src/elements/content-sidebar/activity-feed/annotations/__tests__/AnnotationActivityMenu.test.js

```diff
@@ -31,8 +31,25 @@ describe('elements/content-sidebar/ActivityFeed/annotations/AnnotationActivityMe
         expect(wrapper.exists('[data-testid="edit-annotation-activity"]')).toBe(true);
     });
 
+    test('should render the resolve annotation activity menu item if canResolve is true for unresolved item', () => {
```

Comment:
Should we have a test that validates neither option is shown if `can_resolve` is false?